### ISSUE TITLE
Add options for resuming playlist when interrupt is on

### ISSUE THREAD

**Issue by robbiet76** [whitesoup12/remote-falcon-tracker#124]
Currently, when the interrupt schedule setting is set to "on" and a request is made, the current sequence is immediately interrupted and the new sequence plays.  When the requested sequence finishes, the original sequence resumes where it left off.  Would it be possible to add a setting that EITHER skips the interrupted sequence OR starts it over from the beginning?  I don't like having the viewer see a partial sequence as it seems as if something is wrong with the show.  Also, who wants to see half a song?

In short, I love the ability to interrupt the current running sequence, but don't want to start a partial sequence after a request has finished.
